data-prepper-plugins/opensearch/src/test/java/com/amazon/dataprepper/plugins/sink/opensearch/index/IndexManagerFactoryTests.java

```diff
@@ -29,20 +30,20 @@ public void setup() {
     public void getIndexManager_TraceAnalyticsRaw() {
         final IndexManager indexManager =
                 indexManagerFactory.getIndexManager(IndexType.TRACE_ANALYTICS_RAW, restHighLevelClient, openSearchSinkConfiguration);
-        assertEquals(TraceAnalyticsRawIndexManager.class, indexManager.getClass());
+        assertTrue(indexManager instanceof IndexManager);
     }
 
     @Test
     public void getIndexManager_TraceAnalyticsServiceMap() {
         final IndexManager indexManager =
                 indexManagerFactory.getIndexManager(IndexType.TRACE_ANALYTICS_SERVICE_MAP, restHighLevelClient, openSearchSinkConfiguration);
-        assertEquals(TraceAnalyticsServiceMapIndexManager.class, indexManager.getClass());
+        assertTrue(indexManager instanceof IndexManager);
```

Comment:
You can use from Hamcrest here:

```
assertThat(indexManager, instanceOf(IndexManager.class));
```

There are similar lines below.

Reply:
Just curious, if the JUnit provides the ability, why should we introduce another dependency, Hamcrest? assertThat is not as readable as assertEquals.

Reply:
Sounds good!